src/node/utils/Minify.js

```diff
@@ -178,98 +138,83 @@ const minify = (req, res) => {
 
   const contentType = mime.lookup(filename);
 
-  statFile(filename, (error, date, exists) => {
-    if (date) {
-      date = new Date(date);
-      date.setMilliseconds(0);
-      res.setHeader('last-modified', date.toUTCString());
-      res.setHeader('date', (new Date()).toUTCString());
-      if (settings.maxAge !== undefined) {
-        const expiresDate = new Date(Date.now() + settings.maxAge * 1000);
-        res.setHeader('expires', expiresDate.toUTCString());
-        res.setHeader('cache-control', `max-age=${settings.maxAge}`);
-      }
+  const [date, exists] = await statFile(filename, 3);
```

Comment:
Does it matter that we dropped error handling here?

Reply:
Error handling is still done—it's just done by Express instead of manually ourselves. An advantage of letting Express handle errors: If `NODE_ENV=development` then the 500 page will show a stack trace.